CHANGELOG.md

```diff
@@ -0,0 +1,356 @@
+## Changelog
+
+### Version 1.31
+Oct 22, 2019
+
+-   Update ssh-slaves ([PR-100](https://github.com/jenkinsci/durable-task-plugin/pull/100){.external-link})
+-   Do not fail tests when run on a machine without Docker installed.
+    ([PR-101](https://github.com/jenkinsci/durable-task-plugin/pull/101){.external-link})
+-   Improve watcher logging
+    ([PR-102](https://github.com/jenkinsci/durable-task-plugin/pull/102){.external-link})
+-   Refactor UNIX unit tests for greater test coverage
+    ([PR-103](https://github.com/jenkinsci/durable-task-plugin/pull/103){.external-link})
+-   Allow setting pwsh as Powershell executable
+    ([PR-112](https://github.com/jenkinsci/durable-task-plugin/pull/111){.external-link})
+-   Bugfix: Use setsid instead of nohup
+    - For *NIX systems only, the shell wrapper has been replaced with a pre-compiled golang binary.
+    - The binary launches the script under a new session to better survive unexpected Jenkins terminations.
+    - Just like how the shell wrapper executes in the background (since 1.30), the script launcher
+      is a daemonized process. The means that there is an expectation of orphaned-child cleanup
+      (i.e. zombie-reaping) within the underlying environment.
+    - The binary itself is ~2.5MB per binary. There are 4 pre-compiled binaries (32 and 64bit versions
+      for UNIX and DARWIN).
+    - The memory footprint is ~800KB heavier than the shell wrapper.
+        - The two shell processes (610-640KB) and single sleep process (548KB) are replaced by
+          a single process (~2560KB)
+
+### Version 1.30
```

Comment:
There seems to be some formatting here

Reply:
@car-roll The github diff still shows it as italicised

Reply:
I think the italicised is just how github is showing the raw md file. In the file view, everything looks fine:
https://github.com/jenkinsci/durable-task-plugin/blob/90dc80d05db44255b9b8283c971064bc051b487a/CHANGELOG.md